File: Programming/Perl/S1_Basic/S1_01_Basic_syntax.pl
#!/usr/bin/perl
#   Basic syntax

#T# variable declaration and initialization, identifiers
$var1 = 5;

#T# function call, print to the terminal
print "Hello, world. Var: $var1\n";

#T# Double quotes interpolate its content, single quotes do not
print 'Var: $var1\n';

#T# plain old documentation (pod)
=begin comment
    This multi line comment begins with =begin and ends with =end.
    The pod section ends with =cut
=end comment
=head1 First header
    Text here is under the header 1. Headers can be made up to =head4
=over 4
=item item in list, ends with =back
=item second item
=item third item
=back
=cut

#T# here documents
$text1 = <<"EOF";
    FirstLine
    SecondLine Word3
    Line with var, $var1
EOF
print("\nWholeText:\n$text1");

File: Programming/PHP/S1_Basic/S1_01_Basic_syntax.php
<!--
    Basic syntax
-->


<?php 
//T// Escaping to php with php tags "<?php ? >"
    echo "Hello, World!";

//T// Here document
    print <<<EOF
    FistLine secondWord.
    SecondLine.
EOF;

//T// Variable assignment
    $var1 = 5 + 3;

//T// a block of code goes between curly brackets {}
    {
        print " Val: $var1";
    }
?>